### DISCUSSION TITLE
Send metadata (including screenshots) when publishing app to the Microsoft Store

### DISCUSSION THREAD

**Discussion by mregnauld** [codemagic-ci-cd/codemagic-docs#2669] [Ideas]
### Use case

```Text
My app is available on iOS, macOS, Android and Windows.
For the first 3 platforms, I can publish my app and the associated metadata (description, release notes, ... and the screenshots).

Even though I can't publish the metadata only using Codemagic, I still can do that pretty easily thanks to Fastlane.
```

### Idea

```Text
Since Fastlane is not available for the Microsoft Store (as far as I know), it would be very useful then to have a way to upload all the metadata, including screenshots, to the Microsoft Store, when publishing an app.

It would very useful in at least one of the two situations below:
- the screenshots change frequently (something that is quite common in the early stage of the app)
- the app is available in multiple languages
```

**Comment by mikhail-tokarev** [comment]
Hi @mregnauld 
not sure I understand the idea... Are you looking for general way to use Codemagic for publishing to Microsoft Store? For that we have documentation how to do it https://docs.codemagic.io/yaml-publishing/microsoft-store/

or is it your question more about how to improve existing integration to include screenshots? Do you know if [Microsoft Store API](https://learn.microsoft.com/en-us/windows/uwp/monetize/create-and-manage-submissions-using-windows-store-services) provides this option?

**Reply by mregnauld** [reply]
It's about how to improve the current integration.
I already know how to publish an app to the Microsoft Store using Codemagic, but only the app itself.
What I would like is a way to also update metadata (description, release notes, screenshots, ...) using a Codemagic workflow.

**Reply by mikhail-tokarev** [reply]
thanks for the info. I'll keep it open to see if we have more users asking about this feature
